test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -1,13 +1,12 @@
-#include <gtest/gtest.h>
```

Comment:
OK, it seems clang-format changed its behavior for some reason.
Same reorg happens to me as well if I run it.